### ISSUE TITLE
Honeypot failure with accessibility

### ISSUE THREAD

**Issue by nickdgreen** [elementor/elementor#16021]
When a Honeypot is added to a form a text field of size 1 is added but the field does not contain the <label>***</label> tag. The label tag is required to comply with Web Accessibility Evaluation Tools. The Lable field should be added.

**Comment by mcrowell** [comment]
+1 Good catch!

**Comment by shilo-ey** [comment]
Thanks, what does *** means? `<label>`?

**Comment by nickdgreen** [comment]
Ah, I forgot that Github strips out the html stuff. The three stars were between the "label" and "\lable" tags

**Comment by nickdgreen** [comment]
So that should read  "the field does not contain the `<label>***</lable>` tags

**Comment by tomslominski** [comment]
Would be nice to see a fix, but in the meantime, he's a workaround:

```
/**
 * Add a generic label to honeypot fields.
 *
 * @param array $item
 * @return array
 */
add_action( 'elementor_pro/forms/render/item', function( $item ) {
	if ( 'honeypot' === $item['field_type'] ) {
		$item['field_label'] = __( 'Please do not fill this field out', 'domain' );
	}

	return $item;
}, 11 );
```

**Comment by Tison** [comment]
I'm still seeing this issue as of 11/8/22.  Unfortunately the workaround by @tomslominski does not work in our case.
